test/jvm/extensions/string_extensions_test.rb

```diff
@@ -74,4 +76,39 @@ def test_string_matches_wrong_type
       compile("def match_wrong_type; 'abcdef' =~ 'd'; end")
     end
   end
+  
+  def test_string_to_int_invalid_number
+    cls, = compile(<<-EOF)
+      puts 'abc'.to_int
+    EOF
+    assert_raise_java Java::JavaLang::NumberFormatException do
+      cls.main nil
+    end
```

Comment:
I think we're asserting exception on running main w/o args often enough that we should have an custom assertion method for this. Not necessarily for this PR, but what do you think?